src/Controller/CaseController.php

```diff
@@ -633,4 +646,33 @@ public function move(BoardRepository $boardRepository, CaseEntity $case, Request
             'case' => $case,
         ]);
     }
+
+    /**
+     * @Route("/{id}/delete", name="case_delete", methods={"POST"})
+     */
+    public function delete(CaseEntity $case, Request $request): Response
+    {
+        $this->denyAccessUnlessGranted('edit', $case);
```

Comment:
The voter should probably support a `'delete'` attribute (which may be handled exactly like `'edit'`).

Reply:
Sure - it wouldn't be surprising if this needed some special handling later!